src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManagerInterface.php

```diff
@@ -126,11 +126,22 @@ public function addFormatsAndUrl(Media $media);
     /**
      * Returns download url for given id and filename.
      *
-     * @param string $id
+     * @param string|int $id
      * @param string $fileName
-     * @param string $version
+     * @param string|int $version
      *
      * @return string
      */
     public function getUrl($id, $fileName, $version);
+
+    /**
+     * Returns admin download url for given id and filename.
+     *
+     * @param string|int $id
+     * @param string $fileName
+     * @param string|int $version
+     *
+     * @return string
+     */
+    public function getAdminUrl($id, $fileName, $version);
```

Comment:
Since we have been very careful about BC breaks in #5439 we probably also should not add this method to the interface, because this is a BC break as well.